src/sagemaker/serve/model_server/djl_serving/prepare.py

```diff
@@ -34,27 +34,57 @@
 
 
 def _has_serving_properties_file(code_dir: Path) -> bool:
-    """Placeholder Docstring"""
+    """Check for existing serving properties in the directory"""
     return code_dir.joinpath(_SERVING_PROPERTIES_FILE).is_file()
 
 
-def _members(resources: object, depth: int):
-    """Placeholder Docstring"""
-    for member in resources.getmembers():
-        member.path = member.path.split("/", depth)[-1]
-        yield member
+def _move_to_code_dir(js_model_dir: str, code_dir: Path):
+    """Move DJL Jumpstart resources from model to code_dir"""
+    js_model_resources = Path(js_model_dir).joinpath("model")
+    for resource in js_model_resources.glob("*"):
+        try:
+            shutil.move(resource, code_dir)
+        except shutil.Error as e:
+            if "already exists" in str(e):
+                continue
+
+
+def _extract_js_resource(js_model_dir, js_id: str):
+    """Uncompress the jumpstart resource"""
```

Comment:
can we be consistent with the typing for JumpStart?

Reply:
added typing